styles/archery-styles.css

```diff
@@ -160,6 +160,8 @@ footer p{
 
 .span_1_of_3 img{
   width: 90%;
+  max-height: 200px;
+  object-fit: cover;
```

Comment:
I did some research on "object-fit" and it is awesome! Thanks for mentioning it :)